hardhat.config.js

```diff
@@ -30,33 +30,18 @@ module.exports = {
     dependencyCompiler: {
         paths: [
             '@openzeppelin/contracts/token/ERC20/presets/ERC20PresetFixedSupply.sol',
-            '@openzeppelin/contracts/mocks/ERC20PermitMock.sol',
+            '@josojo/zkevm-contracts/contracts/mocks/ERC20PermitMock.sol',
```

Comment:
I guess there's a reason why this one has to come zkevm-contracts and everything else can come from openzeppelin?

Reply:
I am not sure how this used to work before...

I ma just getting:
Error HH404: File @openzeppelin/contracts/mocks/ERC20PermitMock.sol, imported from contracts/hardhat-dependency-compiler/@openzeppelin/contracts/mocks/ERC20PermitMock.sol, not found.